### PR TITLE
VVS 1 - QueueJob with single column job identifier

### PR DESCRIPTION
**Pull request by mb-dbc-dk**
Old columns synthesized from new column, new column synthesized from old.
Tests ran with testing against old columns.
Test runs with old columns ad nullable with no value.

### CODE REVIEW

File: queue/src/main/java/dk/dbc/search/solrdocstore/queue/QueueJob.java [hunk 1]
```diff
@@ -32,35 +34,76 @@
...
+        return m.group(3);
+    }
+
+    public boolean isWork() {
+        return jobId.startsWith("work:");
+    }
+
+    public String getWork() {
+        if (!jobId.startsWith("work:"))
+            throw new IllegalStateException("Trying to get wotk from jobId: " + jobId);
```
Comment (noah-dbc) [inline queue/src/main/java/dk/dbc/search/solrdocstore/queue/QueueJob.java:105]: trykfejl

**Review by noah-dbc — APPROVED** [review]

File: queue/src/main/java/dk/dbc/search/solrdocstore/queue/QueueJob.java (hunk 1 above)
Comment (mb-dbc-dk) [reply queue/src/main/java/dk/dbc/search/solrdocstore/queue/QueueJob.java:105]: :grin: - en eller anden fra StarWars

File: postgresql/src/main/docker/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM docker.dbc.dk/dbc-postgres-no-auto:10
 MAINTAINER os-team
 
-LABEL SOLR_DOC_STORE_QUEUE_NAMES="Comme seperated list of default queue names (optional)";
+LABEL SOLR_DOC_STORE_QUEUE_NAMES="Comma seperated list of default queue names (optional)";
```
Comment (sahu-dbc-dk) [inline postgresql/src/main/docker/Dockerfile:4]: separated :)

**Review by sahu-dbc-dk — APPROVED** [review]

File: postgresql/src/main/docker/Dockerfile (hunk 2 above)
Comment (mb-dbc-dk) [reply postgresql/src/main/docker/Dockerfile:4]: :stop_sign: - i ved godt jeg ikke kan stave :grinning:

**Pull request merged**